dashboards/llm-d/llm-d.json

```diff
@@ -0,0 +1,2288 @@
+          "id": "",
+          "xyChart": {
+            "chartOptions": {
+              "displayHorizontal": false,
+              "mode": "COLOR",
+              "showLegend": false
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "legendTemplate": "",
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "outputFullDuration": false,
+                  "prometheusQuery": "sum by (role, pod) (\n1 / (count by (cluster, pod, node, role) ({\"__name__\"=\"vllm:generation_tokens_total\"}))\n* on (node) group_left(zone) (\n    label_replace(\n        sum by (metadata_system_name, zone)(\n            avg_over_time({\"__name__\"=\"compute.googleapis.com/instance/gpu/throughput_tx_bytes\",\"monitored_resource\"=\"gce_instance\"}[${__interval}]) / ${__interval}\n        ),\n        \"node\",\n        \"$1\",\n        \"metadata_system_name\",\n        \"(.+)\"\n    )\n))",
+                  "unitOverride": "By"
+                }
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
+      },
+      {
+        "yPos": 4,
+        "xPos": 24,
+        "height": 14,
+        "width": 24,
+        "widget": {
+          "title": "Throughput RX Bytes per Pod",
+          "id": "",
+          "xyChart": {
+            "chartOptions": {
+              "displayHorizontal": false,
+              "mode": "COLOR",
+              "showLegend": false
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "legendTemplate": "",
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "outputFullDuration": false,
+                  "prometheusQuery": "sum by (role, pod) (\n1 / (count by (cluster, pod, node, role) ({\"__name__\"=\"vllm:generation_tokens_total\"}))\n* on (node) group_left(zone) (\n    label_replace(\n        sum by (metadata_system_name, zone)(\n            avg_over_time({\"__name__\"=\"compute.googleapis.com/instance/gpu/throughput_rx_bytes\",\"monitored_resource\"=\"gce_instance\"}[${__interval}]) / ${__interval}\n        ),\n        \"node\",\n        \"$1\",\n        \"metadata_system_name\",\n        \"(.+)\"\n    )\n))",
+                  "unitOverride": "By"
+                }
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
+      },
+      {
+        "yPos": 18,
+        "height": 13,
+        "width": 48,
+        "widget": {
+          "title": "Pod and Node Mapping",
```

Comment:
is there a clearer title for this? I can't tell what the value column in the table like "2.7%" means from this